storage/blockdevice/COMPONENT_QSPIF/source/QSPIFBlockDevice.cpp

```diff
@@ -17,15 +17,15 @@
 
 #include "blockdevice/internal/SFDP.h"
 #include "platform/Callback.h"
-#include "QSPIFBlockDevice.h"
+#include "QSPIF/QSPIFBlockDevice.h"
```

Comment:
"QSPIFBlockDevice: Use fully-qualified include path" doesn't have a "why" explanation in the commit message. We should explain why this is good, or why we need to do it.